.circleci/config.yml

```diff
@@ -73,7 +69,7 @@ linux: &linux
 mac: &mac
   working_directory: ~/SalesforceMobileSDK-Package
   macos:
-    xcode: "9.4.1"
+    xcode: "10.0.0"
```

Comment:
`10.1.0` is out.

Reply:
@brandonpage Can we use 10.1.0 on CircleCI?